src/common/styles/fonts.scss

```diff
@@ -66,3 +66,10 @@ $semiBoldFontFamily: make-font-family($semiBoldFonts) !default;
     font-size: 17px;
     line-height: 24px;
 }
+
+@mixin text-style-body-m {
```

Comment:
The 'body' in the name seems odd. I notice we already have `text-style-title-s` which makes me thing its for a title. Should this new `mixin` be called `text-style-label-m` instead?

Reply:
The `body-m` naming here matches the names of the text styles in our typography figma